modules/ballerina-core/pom.xml

```diff
@@ -66,25 +74,21 @@
                     <groupId>org.ops4j.pax.logging</groupId>
                     <artifactId>pax-logging-api</artifactId>
                 </exclusion>
+                <exclusion>
+                    <groupId>org.wso2.carbon</groupId>
+                    <artifactId>org.wso2.carbon.kernel.feature</artifactId>
+                </exclusion>
+                <exclusion>
+                    <groupId>org.wso2.carbon</groupId>
+                    <artifactId>org.wso2.carbon.runtime.feature</artifactId>
+                </exclusion>
             </exclusions>
         </dependency>
         <dependency>
-            <groupId>org.wso2.carbon.config</groupId>
-            <artifactId>org.wso2.carbon.config</artifactId>
-        </dependency>
-        <dependency>
-            <groupId>org.wso2.carbon.secvault</groupId>
-            <artifactId>org.wso2.carbon.secvault</artifactId>
-        </dependency>
-        <dependency>
-            <groupId>org.wso2.orbit.org.antlr</groupId>
-            <artifactId>antlr4-runtime</artifactId>
+            <groupId>com.fasterxml.jackson.core</groupId>
```

Comment:
Why is this dependency added (jackson-core), this comes as a transitive dependency for jackson-databind. Also, with this change, there will be a build failure, since this dependency version is not mentioned anywhere. If we are adding this, it has to be fixed in ballerina-parent.

Reply:
Added because I felt it's better to have that explicitly rather than depending on a transitive one, it is added to parent with pull - https://github.com/ballerinalang/ballerina-parent/pull/55

Reply:
Yeah, true enough, thanks.